libdnf/dnf-sack.cpp

```diff
@@ -2335,7 +2335,7 @@ void dnf_sack_filter_modules_v2(DnfSack * sack, DnfModulePackageContainer * modu
     }
 
     auto defaultStreams = moduleDefaults.getDefaultStreams();
-    enableModuleStreams(*moduleContainer, install_root);
+    //enableModuleStreams(*moduleContainer, install_root);
```

Comment:
I just want to confirm this is intentional and not something you commented for debug purposes.

Reply:
It's intentional. enableModuleStreams is made obsolete with the Persistor.